app/dao/fact_billing_dao.py

```diff
@@ -468,7 +466,9 @@ def fetch_billing_data_for_day(process_day, service_id=None, check_permissions=F
     return transit_data
 
 
-def _query_for_billing_data(table, notification_type, start_date, end_date, service):
+def _query_for_billing_data(notification_type, start_date, end_date, service):
+    table = NotificationAllTimeView
```

Comment:
i appreciate this keeping the diff short and the review simple, however, if I were writing an SQL query from scratch I'd use the class name each time we reference a column.

```python
return db.session.query(
    NotificationAllTimeView.template_id,
    ...
```

how do you feel about making that change in a separate commit? i don't think this is a 100% necessary change that blocks this PR, but it might be nice to pre-empt us having to stop to think "now what `table` are we talking about here" if we miss a definition that might be many lines up outside of the big sql query definition we're looking at.

Reply:
Sure, changed in https://github.com/alphagov/notifications-api/pull/3546/commits/8e837cf681ea1e02bcc1f87364c16a6b534e22bb.

I had mixed opinions about this myself, so happy to change it if you prefer the direct class version.